MMUL-BASH-01/Exercises/Day-2/Exercise.006

```diff
@@ -0,0 +1,38 @@
+# Exercise 006 - Create a systemd custom service
+
+1) Log into the machine with the credentials you own;
+
+2) Create a file in your homedir called service_action;
+
+3) Edit the file as follow:
+   ```
+   AUTHOR="**INSERT YOUR USERNAME HERE**"
+   
+   while true
+   do
+   echo "Example service from author: ${AUTHOR}"
+   sleep 5;
+   done
+   ```
+
+4) Create a service unit named $USER_exercise.service with this content:
+   ```
+   [Unit]
+   Description=*INSERT YOUR USERNAME HERE*
+
+   [Service]
+   Type=simple
+   ExecStart=/bin/bash /home/*INSERT YOUR USERNAME HERE*/service_action
+
+   [Install]
+   WantedBy=multi-user.target
+   ```
+
+5) Change the mode of the newly created service so that the file can be
+   written and read by owner, read by group, read by other;
+
+6) Copy the newly created service in /etc/systemd/system/
+
+7) With your user, start the newly created service and query its status;
```

Comment:
If you don't want to specify the student must use ```sudo``` to perform the operation, you must omit the "With your user" part (see point 6 as reference); that incipit can land to misunderstanding if students try to start and querying the units without superuser permissions

Reply:
As point 6, fixed point 7.